mdoc/Test/en.expected-eii-implementation-ecmadoc/CustomNamespace/ClassEnumerator.xml

```diff
@@ -0,0 +1,116 @@
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; ClassEnumerator();" />
+      <MemberSignature Language="C++ CX" Value="public:&#xA; ClassEnumerator();" />
+      <MemberSignature Language="C++ WINRT" Value=" ClassEnumerator();" />
+      <MemberSignature Language="ILAsm" Value=".method public specialname rtspecialname instance void .ctor() cil managed" />
+      <MemberSignature Language="JavaScript" Value="function ClassEnumerator()" Usage="var classEnumerator = new ClassEnumerator();" />
+      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
+      <MemberType>Constructor</MemberType>
+      <AssemblyInfo>
+        <AssemblyVersion>0.0.65535.65535</AssemblyVersion>
+      </AssemblyInfo>
+      <Parameters />
+      <Docs>
+        <summary>ECMA ctor summary</summary>
+        <remarks>ECMA ctor remarks</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="CustomNamespace.CustomInterface.Prop1" ExplicitInterfaceMemberName="CustomProp1">
+      <MemberSignature Language="C#" Value="int CustomNamespace.CustomInterface.Prop1 { get; set; }" />
+      <MemberSignature Language="C++ CLI" Value="property int CustomNamespace::CustomInterface::Prop1 { int get(); void set(int value); };" />
+      <MemberSignature Language="C++ CX" Value="property int CustomNamespace::CustomInterface::Prop1 { int get(); void set(int value); };" />
+      <MemberSignature Language="F#" Value="" Usage="CustomNamespace.CustomInterface.Prop1" />
+      <MemberSignature Language="ILAsm" Value=".property instance int32 CustomProp1" />
+      <MemberSignature Language="VB.NET" Value=" Property Prop1 As Integer Implements CustomInterface.Prop1" />
+      <MemberType>Property</MemberType>
+      <AssemblyInfo>
+        <AssemblyVersion>0.0.65535.65535</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Int32</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>ECMA CustomProp1 summary</summary>
+        <value>ECMA CustomProp1 value</value>
+        <remarks>ECMA CustomProp1 remarks</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="CustomNamespace.CustomInterface.Prop2" ExplicitInterfaceMemberName="CustomProp2">
+      <MemberSignature Language="C#" Value="object CustomNamespace.CustomInterface.Prop2 { get; }" />
+      <MemberSignature Language="C++ CLI" Value="property System::Object ^ CustomNamespace::CustomInterface::Prop2 { System::Object ^ get(); };" />
+      <MemberSignature Language="C++ CX" Value="property Platform::Object ^ CustomNamespace::CustomInterface::Prop2 { Platform::Object ^ get(); };" />
+      <MemberSignature Language="F#" Value="" Usage="CustomNamespace.CustomInterface.Prop2" />
+      <MemberSignature Language="ILAsm" Value=".property instance object CustomProp2" />
+      <MemberSignature Language="VB.NET" Value=" ReadOnly Property Prop2 As Object Implements CustomInterface.Prop2" />
```

Comment:
Same question here ... looks like the VB.NET signature should be using `CustomProp2`